#7 contagem_palavras/2018-2/Michel_Araujo/contagem_palavras_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'minitest/autorun'
+require_relative 'frase'
+
+class ContagemPalavrasTest < Minitest::Test
+  def test_conta_uma_palavra
+    skip
```

Comment:
@mikhaelangelo, você precisar remover o `skip` para os testes serem executados, caso contrário, ele 'pula' para o próximo.

Reply:
Sim sim, to ligado. Eu to deixando o skip em todas as funções e só comentando a linha do "skip" quando quero testar. Daí to testando 1 por 1 (comento a linha do skip na função que desejo testar > se o teste passar eu "descomento" a linha para considerar novamente o skip e pulo pra próxima função... Daí por isso acabei enviando esse arquivo de testes com os skips. Mas a classe está correta então eu só deletei o arquivo de testes desse commit.

Reply:
@mikhaelangelo vou fazer a mesclagem (`merge`) e conferir.
Pull request aceito e fechado.

Reply:
@mikhaelangelo, no final, antes de enviar você está então executando seu código em todos os testes ao mesmo tempo, certo? Você vai removendo skip por skip, até não ter mais nenhum.

Reply:
> @mikhaelangelo, no final, antes de enviar você está então executando seu código em todos os testes ao mesmo tempo, certo? Você vai removendo skip por skip, até não ter mais nenhum.

Isso. Executo 1 por 1 e quando chego no último teste, com ele passando, eu rodo novamente mas executando todos (sem nenhum skip).